src/services/storage/utils.ts

```diff
@@ -0,0 +1,40 @@
+import { SupportedTokens } from '../../definitions'
+import config from 'config'
+
+export const WEI = 1000000000000000000
```

Comment:
```suggestion
export const WEI = 1e18
```
We can use the exponent notation to be better readable ;-)

Reply:
Done